src/vaadin-overlay.html

```diff
@@ -957,6 +1055,11 @@
         return active;
       }
 
+      /**
+       * @param {!Element} node
```

Comment:
Just a thought. Technically this should work fine with the more generic `Node` too but I guess it's fine to keep it as is and limit this to `{!Element}` since it is only used in one place where it always gets an `Element`.

Reply:
Changed.